fdbserver/BlobGranuleValidation.actor.cpp

```diff
@@ -570,10 +582,19 @@ ACTOR Future<Void> checkFeedCleanup(Database cx, bool debug) {
 				stableTimestamp = now();
 			}
 			lastGranules = granules;
+			if (g_network->isSimulated()) {
+				if (g_simulator->speedUpSimulation && !speedUpSimTimestamp.present()) {
+					speedUpSimTimestamp = now();
+				}
+			} else {
+				speedUpSimTimestamp = 0.0;
```

Comment:
Yeah essentially the behavior i wanted to add was, if you're in simulation, only trigger this assert if you've been in speed up simulation for some amount of time, in addition to the existing criteria.
This would mean that outside of simulation, behavior is unchanged (which is why i set the timestamp so that part of the check is effectively trivial). Although we currently only run this in simulation anyway so it's not a necessary thing, just nice to have